client/src/app/components/tool-container/tool-header/tool-header.component.scss

```diff
@@ -0,0 +1,26 @@
+@import "../../../../assets/sass/variables";
+
+header {
+    #revisionSelect {
+        color: #000000;
+        margin-right: 40px;
+        width: 100px;
+    }
+
+    button {
+        width: 8%;
+    }
+
+    #guiJsonButtons {
```

Comment:
Likewise, avoid using ids as CSS selectors. Because they are unique, any [qualifier before an id is superfluous](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Writing_efficient_CSS#Guidelines_for_Efficient_CSS) and makes matching selector to element slower. Though now that I think about it, it would be pretty useful to apply everything in this guide to all our CSS. We don't have much of it, but it will grow.